test/e2e_node/testdoc/script.py

```diff
@@ -0,0 +1,190 @@
+"""
```

Comment:
I think this should just be a `#` style comment. Copy the style of other python files in this repo.